ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -1011,6 +1011,17 @@ tests:
     cluster_profile: equinix-ocp-metal
     workflow: agent-e2e-ha-dualstack
   timeout: 6h0m0s
+- as: e2e-agent-5control-ipv4
+  cluster: build05
```

Comment:
shouldn't be optional this job?

Reply:
Yes, it should be optional. Thanks for catching it.